File: src/php/Rpc/Services/UserCenter/Libraries/MemberLib.php
<?php
/**
 * Created by PhpStorm.
 * User: zhangyubo
 * Date: 2019/4/25
 * Time: 11:39
 */

namespace Cecd\Sdk\Rpc\Services\UserCenter\Libraries;

use Cecd\Sdk\Rpc\Services\RpcInterface;

/**
 * Interface MemberLib
 * @package Lumen\Thrift\Rpc\Modules\UserCenter\Libraries
 * @classpath(\App\Libraries\MemberLib)
 */
interface MemberLib extends RpcInterface
{
    public function getSimpleMemberById(int  $member_id) : array ;
}

File: src/java/src/main/java/com/cecd/sdk/examples/usercenter/Libraries/MemberLib.java
package com.xmi01.thriftrpc.examples.usercenter.Libraries;

import com.xmi01.thriftrpc.examples.usercenter.entity.Member;
import com.xmi01.thriftrpc.rpc.RpcDoc;
import com.xmi01.thriftrpc.rpc.RpcInterface;
import com.xmi01.thriftrpc.rpc.exceptions.CeRpcException;

@RpcDoc("App\\Libraries\\MemberLib")
public interface MemberLib extends RpcInterface {
    public Member getSimpleMemberById(int member_id) throws CeRpcException;
}
